utils/utils_test.go

```diff
@@ -0,0 +1,302 @@
+package utils
```

Comment:
On further review, there's no need for this package to begin with. The functions aren't really being called with the sole exception of pointers and those are fungible with ptr.To. The right solution is to remove the package entirely.